src/test/functional/com/hltech/pact/gen/testfeignclient/TestDto.groovy

```diff
@@ -0,0 +1,114 @@
+package com.hltech.pact.gen.testfeignclient
+
+class TestDto {
+
+    private final boolean booleanField
+    private final Boolean booleanClassField
+    private final short shortField
+    private final Short shortClassField
+    private final int intField
+    private final Integer integerClassField
+    private final long longField
+    private final Long longClassField
+    private final double doubleField
+    private final Double doubleClassField
+    private final float floatField
+    private final Float floatClassField
+    private final String stringField
+    private final char charField
+    private final DateTimeDto dateTimeDto
+
```

Comment:
we can add another String field, this time annotated with @JsonProperty and default value